app/views/posts/index.html.erb

```diff
@@ -12,7 +12,8 @@
   <tbody>
     <% @posts.each do |post| %>
       <tr>
-        <td><%= link_to 'Show', post %></td>
+        <td><%=post.post_content%></td>
+        <!-- <td><%= link_to 'Show', post %></td> -->
```

Comment:
what are these comment markers for

Reply:
Potentially could be useful later if we want to drill down on a particular post and for example look at comments